server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/method/PushSubscriptionSetCreatePerformer.scala

```diff
@@ -87,3 +95,20 @@ class PushSubscriptionSetCreatePerformer @Inject()(pushSubscriptionRepository: P
       case (path, _) => SetError.invalidArguments(SetErrorDescription(s"Unknown error on property '$path'"))
     }
 }
+
+object PushSubscriptionSetCreateProcessor {
```

Comment:
I don't like this name, I used it because the issue wrote that (https://github.com/linagora/james-project/issues/4411)
Should we rename it to "PushVerificationProcessor" ? or move it directly inside the `PushSubscriptionSetCreatePerformer` class.